.eslintrc

```diff
@@ -25,6 +25,7 @@
     "react/prop-types": 0,
     "react/sort-prop-types": 0,
     "react/no-string-refs": 0,
+    "react/no-did-update-set-state": 0,
```

Comment:
Needed this so I could move the componentWillReceiveProps methods into componentDidUpdate.